nextjs/run-tests.js

```diff
@@ -65,7 +65,9 @@ async function main() {
   }
 
   console.log('Running tests with concurrency:', concurrency)
-  let tests = process.argv.filter((arg) => arg.endsWith('.test.js'))
+  let tests = process.argv.filter(
+    (arg) => arg.endsWith('.test.js') || arg.endsWith('.test.ts')
```

Comment:
@flybayer could this become a problem when merging Next.js upstream? Should I contribute this to Next.js instead?

Reply:
this is fine